_posts/2019-12-05-elasticsearch-percolator-qualification.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: post
+title:  "Elasticsearch percolator use case for document classification"
+excerpt: ""
+date:   2019-12-05 12:00:00 +0100
+categories: python
+tags: python elasticsearch percolator
+---
+
+Currently at Allo-Media, we are using Elasticsearch in its general workflow which is to create an index and store documents holding our calls metadata, and then allowing to search through these documents given some business criteria like : "Give me all calls from client Acme, where the customer speaks about the french strike".
```

Comment:
Nits: 

- *we use*
- maybe succintly define what a *call* is, or maybe just use *phone call metadata*
- in English there's no space before a colon
- Language names always start with a capital letter, here *the French strike*